lib/oscap_tests.pm

```diff
@@ -13,21 +13,39 @@ use warnings;
 use testapi;
```

Comment:
normally we put this imports after the base or parent, which is usually the first one.

Reply:
Fixed